src/app/components/AdvancedSearch/AdvancedSearch.jsx

```diff
@@ -207,15 +206,32 @@ class AdvancedSearch extends React.Component {
     event.preventDefault();
     event.stopPropagation();
     const fullQuery = this.parseStateToQuery();
+    if (!fullQuery || !fullQuery.queries || fullQuery.queries.length < 1) {
+      this.setState({ error: true, errorMsg: 'Please fill some query' });
+      return;
+    }
     if (fullQuery) {
       this.boundActions.userQuery(fullQuery);
       const path = `/search?${getQueryString(fullQuery)}`;
       this.context.router.push(path);
     }
   }
 
+  validate() {
```

Comment:
Does this method get called anywhere? Seems the functionality is built in at 209-212